lib/src/rules/use_function_type_syntax_for_parameters.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2018, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+import 'package:linter/src/analyzer.dart';
+
+const _desc = r'Use function type syntax for parameters.';
+
+const _details = r'''
+
+**USE** function type syntax for parameters.
```

Comment:
I'm not sure what the convention is here, but I found one rule that says "**CONSIDER** using ...". It also has a name prefixed with "prefer_". I was just surprised to see "CONSIDER" in the description of the PR and "USE" in the description of the rule.

Reply:
Yeah.  We're not super consistent here...  At this point I think it's a matter of taste whether to go with "use" or "prefer" (though maybe we should get more crisp about that).  I guess the important thing is to have the name and advice agree as best as possible and this (now) looks OK to me.